scripts/prepare_cluster.sh

```diff
@@ -222,11 +225,20 @@ for ((i=0; i<${#MASTERS[@]}; i++)); do
         yq e 'select(.kind == "CustomResourceDefinition")' "$OUTPUT_DIR_MASTER/root/manifest-kube-system.yaml" >> "$OUTPUT_DIR_MASTER/root/crds.yaml"
     fi
 
+    if [ $IS_IN_AZURE == "true" ]; then
+        unique_fourth_octet=$(echo $NODE_HOST_IP | cut -d . -f 4)
```

Comment:
Let's try to be consistent :)
```suggestion
        unique_fourth_octet=$(cut -d . -f 4 <<< "$NODE_HOST_IP")
```